File: packages/pensopay_android/android/src/main/kotlin/com/flutter/pensopay/networking/pensopayapi/pensopaylink/models/PPNin.kt
package com.flutter.pensopay.networking.pensopayapi.pensopaylink.models

import org.json.JSONObject

class PPNin: JSONObject() {

    // Optional Properties

    var number: String? = null
    var country_code: String? = null
    var gender: String? = null //TODO: Convert this into the Gender enum

}

File: packages/pensopay_ios/ios/Classes/Networking/Pensopay API/Pensopay Link/Models/PPNin.swift
//
//  PPNin.swift
//  PensopaySDK
//
//  Created on 07/26/2022
//  Copyright © 2022 Pensopay. All rights reserved.
//

import Foundation

public class PPNin: Codable {

    // MARK: - Enums
    
    private enum Gender: String {
        case male
        case female
    }
    
    
    // MARK: - Properties
    
    public var number: String?
    public var country_code: String?
    public var gender: String? //TODO: Convert this into the Gender enum
    
}
